tests/pr_test.go

```diff
@@ -113,3 +112,45 @@ func TestRunAdvancedExampleUpgrade(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func setupOptionsStandardSolution(t *testing.T, prefix string) *testhelper.TestOptions {
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
+		Prefix:        prefix,
+		ResourceGroup: resourceGroup,
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"existing_kms_instance_crn": permanentResources["hpcs_south_crn"],
+		"kms_endpoint_type":         "public",
+		"resource_group_name":       options.Prefix,
+	}
+
+	return options
+}
+
+func TestRunStandardSolution(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsStandardSolution(t, "redis-st-da")
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunStandardUpgradeSolution(t *testing.T) {
+	t.Parallel()
+	t.Skip()
+
+	options := setupOptionsStandardSolution(t, "redis-st-da-upg")
+
+	output, err := options.RunTestUpgrade()
+	if !options.UpgradeTestSkipped {
+		assert.Nil(t, err, "This should not have errored")
+		assert.NotNil(t, output, "Expected some output")
+	}
+}
```

Comment:
Adding two new tests to ICD is too much. IMHO let's drop TestRunAdvancedExampleUpgrade, since there is a DA upgrade; just make sure this is not causing a gap. eg. FSCloud is already pinned to latest. Should either of these be pinned?

Reply:
@shemau If we drop TestRunAdvancedExampleUpgrade, then we won't have any upgrade tests with CBR, which might be a problem.

Reply:
We could remove TestRunRedisFSCloudExample because DA will be calling FScould module and we have added a test for DA.

Reply:
I have moved TestRunRedisFSCloudExample to other_test.go and pinned TestRunStandardSolution to the latest redis version. If anything else is needed, please let me know.